docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc

```diff
@@ -32,12 +32,13 @@ exclude fields. For more information about field selection, refer to the
 
 //tag::classification-algorithms[]
 {classanalysis-cap} uses an ensemble algorithm that is a type of boosting called 
-boosted tree regression model which combines multiple weak models into a 
+extreme gradient boost (XGboost) which combines multiple weak models into a 
```

Comment:
```suggestion
extreme gradient boosting (XGBoost), which combines multiple weak models into a 
```

Reply:
It should say that our ensemble algorithm is **similar to** XGBoost. As the development of our code progressed, we diverged in quite a number of details and implementation decisions from the "original" XGBoost.